public_dropin_gpu_environments/nim_proxy/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM datarobot/dropin-env-base:1.0.0-python-3.11.11-slim-bookworm
+
+RUN useradd -u 1000 -m drum
+
+# install genai-perf tool
+# see https://docs.nvidia.com/nim/benchmarking/llm/latest/step-by-step.html
+RUN mkdir -p /.cache && chown -R drum:drum /.cache
+RUN apt update && apt install -y --no-install-recommends git
+RUN git config --global credential.helper store
+
+RUN apt install -y libstdc++6 gcc g++ make cmake clang
+RUN pip install genai-perf==0.0.10
+
+
+COPY public_dropin_gpu_environments/nim_proxy/* /opt/code/
+RUN pip install -r /opt/code/requirements.txt
+
+# install DRUM in development mode
+COPY custom_model_runner /opt/code/custom_model_runner/
+RUN pip install -e /opt/code/custom_model_runner/
```

Comment:
all this is needed only during development. image will be simplified and should contain minimum dependencies.